build-scripts/build-core-only.mjs

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs/promises';
+import * as path from 'path';
+
+const jsonPath = path.resolve('package.json');
+
+const pkg = JSON.parse(await fs.readFile(jsonPath, 'utf8'));
+pkg.workspaces = pkg.workspaces.filter(w => !w.includes('modules/'));
```

Comment:
Codacy has a fix for the issue: [Replace `w` with `(w)`](https://app.codacy.com/gh/the-convocation/venat/pullRequest?prid=9430453)
```suggestion
pkg.workspaces = pkg.workspaces.filter((w) => !w.includes('modules/'));
```